ls1intum/Artemis
# `Exam mode`: Display working time on student exam detail view correctly

### Checklist
#### General


#### Client

### Motivation, Context and Description
Currently, the working time form on the student exam detail view does not display the minutes correctly (always 0). This happens because the component expects a string format from seconds to string parser that was replaced some time ago. This PR brings the correct format back. 
I have also replaced sinon with jest in the respective test suite.

Currently:
![broken-working-time](https://user-images.githubusercontent.com/45300855/139592022-59145b43-90f3-4ee8-a529-0f998221fa53.PNG)

Fixed:
![broken-working-time-fixed](https://user-images.githubusercontent.com/45300855/139595797-20f1f949-eef7-4ffc-a69a-c59f34010bfe.PNG)

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Create an Exam with a working time of 90 minutes.
2. Create one exercise
3. Register students and generate student exams
4. View the exam for any student
5. Make sure that the individual working time is displayed correctly (1 hour, 30 minutes)


### Review Progress
#### Code Review
#### Manual Tests

### Test Coverage
